tensorflow_encrypted/__init__.py

```diff
@@ -9,8 +11,11 @@
 from . import convert
 
 
+def set_random_seed(seed: int) -> None:
```

Comment:
maybe we just call `tf.set_random_seed` directly if there's nothing more to be done?

Reply:
It depends how far we want to play along the lines of "You don't even need to import Tensorflow".

After this session PR, my goal is to find a way to duplicate all function protocols to the main scope (ie `prot.matmul` -> `tfe.matmul`).
The same work could be done to move all `tf` function to `tfe` scope, do you find that interesting or useless?

Reply:
hmm, not sure. i like the idea of having the same API but maybe holding back a bit since:
- 1) not sure it's priority right now
- 2) we are complaining good integration with ordinary TF, ie in some use cases we might actually want people to import both

Reply:
in any case, i like the idea of `tfe.matmul` for ops we have already a lot